### PR TITLE
fix: script does not work when the route is bound to a service.

### PR DESCRIPTION
**Pull request by tzssangglass**
### What this PR does / why we need it:

fix: #3663
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by tzssangglass** [comment]
@spacewander @membphis CI is ok, but I don't koow why PR title is solecistic

**Comment by Firstsawyou** [comment]
Your PR title needs to look like this: `fix: script does not work when the route is bound to a service`

File: t/admin/script.t [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+repeat_each(1);
+log_level('info');
+no_long_string();
+no_root_location();
+
+run_tests;
+
+__DATA__
+
+=== TEST 1: add service and service has plugin
```
Comment (tokers) [inline t/admin/script.t:28]: Title should be changed since you just add exactly one service.

Comment (spacewander) [reply t/admin/script.t:28]: The title is correct, the service has plugin.

File: t/admin/script.t [hunk 2]
```diff
@@ -0,0 +1,103 @@
+#
```
Comment (spacewander) [inline t/admin/script.t:1]: Can we add this under `t/script` instead?

Comment (tzssangglass) [reply t/admin/script.t:1]: got

**Review by spacewander — APPROVED** [review]

File: t/admin/script.t (hunk 1 above)
Comment (tokers) [reply t/admin/script.t]: I misunderstood it when I first saw it... Maybe `add service which has plugins` is better.

Comment (tzssangglass) [reply t/admin/script.t]: update

**Review by membphis — APPROVED** [review]

**Pull request merged**